### DISCUSSION TITLE
Multicolored brand icons?

### DISCUSSION THREAD

**Discussion by KamilDev** [simple-icons/simple-icons#12479] [General]
From a first glance, is Simple Icons only supporting single solid colored icons? I was hoping for multicolored icons such as Google.
![Google__G__logo](https://github.com/user-attachments/assets/463a35df-ad51-4766-8bbb-49aaabb92602)

If Simple Icons doesn't support multicolored icons, does anyone know of a brand icon library that supports them?

**Comment by adamrusted** [comment]
You're correct, we're a monochrome icon library, and have turned down brands before if they couldn't sensibly (or legally) be converted to monochrome.

I'm not sure on other libraries personally, there seems to be a new icon pack every day at the moment, so I'm sure one will surface for you soon, given you've interacted here! 🤣
